lib/engine/corporation.rb

```diff
@@ -111,6 +111,10 @@ def add_operating_info!(turn, round_num, info)
       @operating_history[[turn, round_num]] = info
     end
 
+    def operating_info(turn, round_num)
```

Comment:
let's delete this and add_operating_info! they are both unneeded

Reply:
i think it's a bit confusing because when we both initially looked at it, add_operating_info! we thought that it was adding to an array, just exposing the hash will be more readable to the user

so maybe change add_operating_info to be

def operating_info(turn, round_num)=
...

Reply:
yep perfect, let's do this